### PR TITLE
Added User Survey Plugin to prepackaged plugins

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Added [User Survey Plugin](https://github.com/mattermost/mattermost-plugin-user-survey/) as a prepackaged plugin

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-60205

#### Release Note
```release-note
Adding mattermost-plugin-user-survey to pre-packaged plugins
```

### CODE REVIEW

**Review by larkox — APPROVED** [review]

**Comment by harshilsharma63** [comment]
I'll update the plugin version here once this PR - https://github.com/mattermost/mattermost-plugin-user-survey/pull/57 is merged and a new plugin version is released.

File: server/Makefile [hunk 1]
```diff
@@ -157,6 +157,7 @@ PLUGIN_PACKAGES += mattermost-plugin-zoom-v1.8.0
 PLUGIN_PACKAGES += mattermost-plugin-ai-v0.8.3
 PLUGIN_PACKAGES += focalboard-v8.0.0
 PLUGIN_PACKAGES += mattermost-plugin-msteams-v2.0.3
+PLUGIN_PACKAGES += mattermost-plugin-user-survey-v1.1.0
```
Comment (marianunez) [inline server/Makefile:160]: This will be bumped to 1.1.1 once the open PR for the bug fix is completed, correct?

Comment (harshilsharma63) [reply server/Makefile]: @marianunez done.

**Review by yasserfaraazkhan — APPROVED** [review]
Verified installing locally by running 'make prepackaged-plugins'

Screenshot 2024-08-22 at 7 41 04 PM

**Comment by harshilsharma63** [comment]
@marianunez we're good to merge this. Please approve and merge this.

cc @amyblais

**Review by marianunez — APPROVED** [review]

**Pull request merged**